Quintor/src/main/java/com/example/quintor/FileDescription.java

```diff
@@ -0,0 +1,61 @@
+package com.example.quintor;
+
+public class FileDescription {
+    private int id;
+
+    private int numberOfDebitEntries;
+
+    private int numberOfCreditEntries;
+
+    private double amountOfDebitEntries;
+
+    private double amountOfCreditEntries;
+
+    public FileDescription(int id, int numberOfDebitEntries, int numberOfCreditEntries, double amountOfDebitEntries, double amountOfCreditEntries) {
+        this.id = id;
+        this.numberOfDebitEntries = numberOfDebitEntries;
+        this.numberOfCreditEntries = numberOfCreditEntries;
+        this.amountOfDebitEntries = amountOfDebitEntries;
+        this.amountOfCreditEntries = amountOfCreditEntries;
+    }
+
+    public int getId() {
+        return id;
+    }
+
+    public void setId(int id) {
+        this.id = id;
+    }
+
+    public int getNumberOfDebitEntries() {
+        return numberOfDebitEntries;
+    }
+
+    public void setNumberOfDebitEntries(int numberOfDebitEntries) {
```

Comment:
Ik weet niet of dit al werk is voor nu, maar misschien hier een check in of de int meer is dan 0?

Reply:
En dan bij de rest van de setters ook checks doen, maar ik weet niet zo goed of da nu al moet?

Reply:
Dit komt later nog